modules/with_cfg.bzl/0.3.0/presubmit.yml

```diff
@@ -0,0 +1,27 @@
+bcr_test_module:
+  module_path: "examples"
+  matrix:
+    platform: ["debian10", "macos_legacy", "ubuntu2004"]
+    bazel: ["6.x", "7.x"]
+  tasks:
+    run_tests:
+      name: "Run test module"
+      platform: ${{ platform }}
+      bazel: ${{ bazel }}
+      test_targets:
+        - "//..."
+    run_windows_tests:
+      name: "Run test module"
+      platform: "windows"
+      bazel: ${{ bazel }}
+      test_targets:
+        - "//..."
+        # Running these tests on Windows requires non-standard Visual Studio
+        # components and tweaks to PATH.
+        - "-//cc_asan_test/..."
+        - "-//cc_asan_test_with_reset/..."
+        # Fails with:
+        # LAUNCHER ERROR: Cannot launch process: "C:/tools/msys64/usr/bin/bash.exe
+        # Reason: (error: 8): Not enough memory resources are available to process this command.
+        - "-//sh/..."
+        - "-//args_location_expansion/..."
```

Comment:
@meteorcloudy I'm running into this issue again and found [this](https://learn.microsoft.com/en-us/windows-hardware/manufacture/desktop/windows-re-troubleshooting-features?view=windows-11#not-enough-memory-resources-error-when-you-open-a-command-prompt-window-in-windows-pe-for-windows-10-version-1803), where the "Not enough memory resources" error is linked to missing system files. Do you have means to test whether the Windows VM image can successfully launch an `sh_test`?

Reply:
I can confirm if I change `my_sh_test` back to `sh_test`, it works as expected (we do run some shell integration tests on Windows for Bazel).

I noticed with `my_sh_test`, it seems the test was in a dead loop which eventually led to the "Not enough memory" error.

Also the bazel-bin output is different:

my_sh_test:
```
$ tree bazel-bin
bazel-bin
└── sh
    ├── test.exe -> /c/tmp/7466aj6n/execroot/_main/bazel-out/x64_windows-fastbuild/bin/sh/test_/test.exe
    ├── test.exe.repo_mapping
    ├── test.exe.runfiles
    │   ├── MANIFEST
    │   ├── _main
    │   │   └── sh
    │   │       ├── test.exe -> /c/tmp/7466aj6n/execroot/_main/bazel-out/x64_windows-fastbuild/bin/sh/test.exe
    │   │       ├── test.sh -> /c/users/pcloudy/workdir/with_cfg.bzl/examples/sh/test.sh
    │   │       └── test_
    │   │           ├── test -> /c/tmp/7466aj6n/execroot/_main/bazel-out/x64_windows-fastbuild/bin/sh/test_/test
    │   │           └── test.exe -> /c/tmp/7466aj6n/execroot/_main/bazel-out/x64_windows-fastbuild/bin/sh/test_/test.exe
    │   └── _repo_mapping -> /c/tmp/7466aj6n/execroot/_main/bazel-out/x64_windows-fastbuild/bin/sh/test.exe.repo_mapping
    ├── test.exe.runfiles_manifest
    └── test_
        ├── test -> /c/users/pcloudy/workdir/with_cfg.bzl/examples/sh/test.sh
        ├── test.exe
        ├── test.exe.repo_mapping
        ├── test.exe.runfiles
        │   ├── MANIFEST
        │   ├── _main
        │   │   └── sh
        │   │       ├── test.sh -> /c/users/pcloudy/workdir/with_cfg.bzl/examples/sh/test.sh
        │   │       └── test_
        │   │           ├── test -> /c/tmp/7466aj6n/execroot/_main/bazel-out/x64_windows-fastbuild/bin/sh/test_/test
        │   │           └── test.exe -> /c/tmp/7466aj6n/execroot/_main/bazel-out/x64_windows-fastbuild/bin/sh/test_/test.exe
        │   └── _repo_mapping -> /c/tmp/7466aj6n/execroot/_main/bazel-out/x64_windows-fastbuild/bin/sh/test_/test.exe.repo_mapping
        └── test.exe.runfiles_manifest
```

sh_test

```
$ tree bazel-bin
bazel-bin
└── sh
    ├── test -> /c/users/pcloudy/workdir/with_cfg.bzl/examples/sh/test.sh
    ├── test.exe
    ├── test.exe.repo_mapping
    ├── test.exe.runfiles
    │   ├── MANIFEST
    │   ├── _main
    │   │   └── sh
    │   │       ├── test -> /c/tmp/7466aj6n/execroot/_main/bazel-out/x64_windows-fastbuild/bin/sh/test
    │   │       ├── test.exe -> /c/tmp/7466aj6n/execroot/_main/bazel-out/x64_windows-fastbuild/bin/sh/test.exe
    │   │       └── test.sh -> /c/users/pcloudy/workdir/with_cfg.bzl/examples/sh/test.sh
    │   └── _repo_mapping -> /c/tmp/7466aj6n/execroot/_main/bazel-out/x64_windows-fastbuild/bin/sh/test.exe.repo_mapping
    └── test.exe.runfiles_manifest
```

Reply:
Thanks for investigating this. It does work in GitHub Actions CI on Windows, which is surprising. The slightly different output layout is expected, but maybe incorrect? Does Bazel CI enable symlinks on Windows by default?

Reply:
> Does Bazel CI enable symlinks on Windows by default?

No, I believe https://github.com/fmeum/with_cfg.bzl/blob/main/examples/.bazelrc#L3 is respected.